test/e2e/fixtures/DatePicker/MomentJalaliDateCalendar.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 import { AdapterMomentJalaali } from '@mui/x-date-pickers/AdapterMomentJalaali';
 import moment from 'moment-jalaali';
-import 'moment/locale/fa';
+import 'moment/dist/locale/fa';
```

Comment:
I guess it has something to do with esm/cjs, it can't find the locale file if we don't use `dist` here :/

There might be better ways of fixing this? I tried messing with the vitest config on what should be compiled vs not, but I couldn't find a solution.

I'll try to inline it again just in case 🫠

Reply:
https://github.com/vitejs/vite/issues/5359
https://github.com/vitejs/vite/issues/3755#issuecomment-860548526

seems like it is a common issue